src/Handlebars/Context.php

```diff
@@ -73,6 +99,18 @@ public function pushIndex($index)
         array_push($this->index, $index);
     }
 
+    /**
+     * Pushes data variables onto the stack. This is used to support @data variables.
+     * @param array $data Associative array where key is the name of the @data variable and value is the value.
```

Comment:
Should include an @throws annotation indicating that a LogicException can be thrown.